### DISCUSSION TITLE
How to get the link to the Edgar filing doc?

### DISCUSSION THREAD

**Discussion by botradingblog** [dgunning/edgartools#157] [Q&A] [answered]
Hi,
First, thank you so much for creating this awesome tool! 
I have a question on if there's a way to get the link to the original Edgar filing document (e.g. 10-Q or 10-K). I would like to add it as a reference. If it's not possible, perhaps this would be a good feature for a future version?

**Comment by dgunning** [comment, marked as answer]
Here's how you can do it
```python
filings = get_filings()
f = filings[1]
f.url
# The homepage url
'https://www.sec.gov/Archives/edgar/data/1999710/0001493152-24-047483-index.html'

# The filing document url
f.document.url
'https://www.sec.gov/Archives/edgar/data/1999710/000149315224047483/xsl1-A_X01/primary_doc.xml'
>>>
```
If you have a TenK object you can access the filing using `_filing` to get access to the filing and then get the urls
```python
>>> filings = get_filings(form="10-Q")
>>> f = filings[1]
>>> tenk = f.obj()
>>> tenk._filing
```
